express-notes-api/server.js

```diff
@@ -0,0 +1,113 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+const app = express();
+app.use(express.json());
+
+async function fileReader() {
```

Comment:
Usually functions are named as verbs since it reads better when you call it. Compare
`const data = await fileReader();`
to
`const data = await readFile();`

(Of course, you can't use `readFile` since it conflicts with fs.readFile`. That's why I used `readData`.)